lumicks/pylake/kymotracker/tests/test_msd.py

```diff
@@ -34,3 +34,27 @@ def test_msd_estimation(time, position, max_lag, lag, msd):
 def test_estimate(idx, y, diffusion_const, weights):
     diffusion_est = estimate_diffusion_constant(idx, y, weights=weights)
     assert np.allclose(diffusion_est, diffusion_const)
+
+
+@pytest.mark.parametrize(
+    "frame_idx, coordinate, time_step, max_lag, diffusion_const",
+    [
+        (np.array([1, 2, 3, 4, 5]), np.array([-1.0, 1.0, -1.0, -3.0, -5.0]), 0.5, 50, 4.53333333),
+        (np.array([1, 2, 3, 4, 5]), np.array([-1.0, 1.0, -1.0, -3.0, -5.0]), 1.0, 50, 2.26666667),
+        (np.array([1, 2, 3, 4, 5]), np.array([-1.0, 1.0, -1.0, -3.0, -5.0]), 1.0, 2, 3.33333333)
+    ]
+)
+def test_estimate(frame_idx, coordinate, time_step, max_lag, diffusion_const):
+    diffusion_est = estimate_diffusion_constant_simple(frame_idx, coordinate, time_step, max_lag)
+    assert np.allclose(diffusion_est, diffusion_const)
```

Comment:
can get rid of the previous `test_estimate()` if `estimate_diffusion_constant` is absorbed into the calling function

Reply:
It's gone! :)